test/Core.Test/Models/Business/OrganizationLicenseFileFixtures.cs

```diff
@@ -111,7 +111,6 @@ public static Organization OrganizationFactory() =>
             SmServiceAccounts = 8,
             MaxAutoscaleSmSeats = 101,
             MaxAutoscaleSmServiceAccounts = 102,
-            LimitCollectionCreationDeletion = true,
             AllowAdminAccessToAllCollectionItems = true,
```

Comment:
You've deleted `LimitCollectionCreationDeletion` but not `AllowAdminAccessToAllCollectionItems`. You can remove both or update it to have all 3 settings (the comment suggests that we should keep them in, but they're excluded from the assertions anyway so it doesn't really matter).